### PR TITLE
Token Endpoint

### PR DESCRIPTION
**Pull request by AbdelilahLbardi**
Changelist

- [x] Add support for `PUT` & `DELETE` to the `HttpLayer`.
- [x] `Token` create endpoint.
- [x] `Token` update endpoint.
- [x] `Token` delete endpoint.
- [x] Documentation update

Breaking changes:
- N/A

### CODE REVIEW

**Review by fosron — APPROVED** [review]
In README examples we should have `$mailersend = new MailerSend(['api_key' => 'key']);`

File: src/Helpers/Builder/TokenParams.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+use JsonSerializable;
+use MailerSend\Exceptions\MailerSendAssertException;
+use MailerSend\Helpers\GeneralHelpers;
+use Tightenco\Collect\Contracts\Support\Arrayable;
+
+class TokenParams implements Arrayable, JsonSerializable
+{
+    private string $name;
+    private string $domainId;
+    private string $token;
```
Comment (doobas) [inline src/Helpers/Builder/TokenParams.php:15]: Is this needed?

**Review by doobas — APPROVED** [review]

File: src/Helpers/Builder/TokenParams.php (hunk 1 above)
Comment (doobas) [reply src/Helpers/Builder/TokenParams.php:15]: `private string $token;` <<--

Comment (AbdelilahLbardi) [reply src/Helpers/Builder/TokenParams.php]: @doobas adjusted on d41ebc0

**Pull request merged**